net/scm/xscm.c

```diff
@@ -27,17 +28,17 @@ enum xaprc00x_state {
 	SCM_UNOPEN=0, /* Initial state, unopened */
 	SCM_ESTABLISHED, /* Connection established */
 	SCM_SYN_SENT, /* Sent a connection request, waiting for ACK */
+	SCM_SYN_RECV, /* A connection request has been responded to but not processed */
```

Comment:
These states are very TCP specific.  (I know we already discussed that, and leaving it as is for now. Just wanted to mention that here for completeness.)